Documentation/teaching/lectures/interrupts.rst

```diff
@@ -842,24 +842,24 @@ low latency requirements and high frequency:
 
       /* PLEASE, avoid to allocate new softirqs, if you need not _really_ high
          frequency threaded job scheduling. For almost all the purposes
-	 tasklets are more than enough. F.e. all serial device BHs et
```

Comment:
My editor must be changing these, is that a problem?

Reply:
yes. we should not modify random lines by adding/removing spaces.

Reply:
can I make another commit on top? or should I edit my previous commits?

Reply:
No, I meant locally. But I see what you mean, I should edit my last commit using `--amend`